### PR TITLE
Hide 'Featured in' section and update blog strip typography'

### LINKED ISSUE #8337: Hide featured in strip from homepage
Per PdM meeting, we are going to move the blogs up by hiding the 'Featured in' strip.

Please just hide with jinja2 comments for now.

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Temporarily hid the 'featured in' section to move the blog posts up
- Updated the typography of the blog posts slightly including adding one more row in the line clamp

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the featured in section is not visible and the blog section is higher.

## Issue / Card

Fixes #8337

## Screenshots

image

### CODE REVIEW

File: templates/base_index.html [hunk 1]
```diff
@@ -63,6 +63,7 @@ <h2>Modern enterprise open&nbsp;source</h2>
     <hr class="p-separator" />
   </div>
 
+  {# hide for now
```
Comment (anthonydillon) [inline templates/base_index.html:66]: Can we remove code commented out code and use version control to retrieve it when we need too?

**Review by anthonydillon — COMMENTED** [review]
Comment about the comment

File: templates/base_index.html (hunk 1 above)
Comment (pmahnke) [reply templates/base_index.html]: @anthonydillon I guess we will disagree about this forever, but I have removed it for you.

**Review by anthonydillon — APPROVED** [review]
LGTM

**Pull request merged**